cs/configuring.texy

```diff
@@ -107,6 +108,28 @@ $configurator->addParameters([
 ]);
 \--
 
+Lze definovat i dynamické parametry, které nezpůsobí přegenerování containeru.
```

Comment:
Nikde není informace o tom, že normální parametry přegenerují container, takže to chce vysvětlit lépe. Zrovna teď mě nenapadá úplně jak.